src/main.zig

```diff
@@ -142,6 +145,50 @@ fn help(name: []const u8, args: *std.process.ArgIterator, allocator: std.mem.All
     try std.io.getStdOut().writer().print("\n", .{});
 }
 
+pub fn init(name: []const u8, args: *std.process.ArgIterator, allocator: std.mem.Allocator) !void {
+    _ = args;
+    _ = name;
+    _ = allocator; // defining these as _ to silence the compiler
+
+    var path_buffer: [std.fs.MAX_PATH_BYTES]u8 = undefined;
+    const main_path = std.fs.cwd().realpath("main.orng", &path_buffer) catch |err| switch (err) {
```

Comment:
![image](https://github.com/user-attachments/assets/3bdad2a3-2ef9-4819-bca9-5f89b3d8df9e)
I'm getting this error whenever I run `orng init` in an empty directory. On second thought, I think `realpath` isn't the best way forward in this particular case. Here's what I think works better instead:

```zig
if (std.fs.cwd().openFile(main_path, .{})) |_| {
    std.log.err("the file `{s}` already exists", .{main_path});
    return error.BuildOrngError;
} else |err| switch (err) {
    error.FileNotFound => {},
    else => return err,
}
```
This tries to open the main file, and if it succeeds, returns the `BuildOrngError`, otherwise it succeeds only if it failed because the file wasn't there. A bit complicated, and maybe a bit much for a first timer, but you're doing a great job so far. It would also be used to check if for the `build.orng` file too of course.